rollup.config.js

```diff
@@ -36,6 +44,9 @@ function getRollupObject ({ minifying, format = 'umd' } = {}) {
             babel()
         ]
     };
+    if (lite) {
+        nonMinified.external = ['estraverse'];
```

Comment:
This should probably just list the keys of the `dependencies` from `package.json`. If we add another dependency, I imagine this build would want to exclude it.

Reply:
Good point, fixed!